R/graphAdd.R

```diff
@@ -3,6 +3,8 @@
 #' @param eList named list with at least the Daily, Sample, and INFO dataframes
 #' @param showArrows logical whether or not to show arrows representing flow segments
 #' @param showRect logical whether or not to show rectangles representing sample segments
+#' @param customPalette character vector of colors as a hexadecimal string of the form "#rrggbb". 
+#' Defaults to NULL, which indicates the use of a default palette (up to 21 segments).
 #' @export
 #' @importFrom RColorBrewer brewer.pal
```

Comment:
You'll need to delete this line, and delete the line from DESCRIPTION (and push the changes to NAMESPACE too)